### DISCUSSION TITLE
Pivot Actions don't appear on resource table rows when showInline() is used

### DISCUSSION THREAD

**Discussion by puzzledmonkey** [laravel/nova-issues#5152] [Ideas & Feature Requests]
- Laravel Version: 9.41.0
- Nova Version: 4.19.4
- PHP Version: 8.1.12
- Database Driver & Version: n/a
- Operating System and Version: n/a
- Browser type and version: n/a
- Reproduction Repository: n/a

### Description:

Pivot Actions annotated with showInline() do now show up on the inline actions menu

### Detailed steps to reproduce the issue on a fresh Nova installation:

Create 2 resources with a BelongsToMany relationship with a pivot action using showInline():

```
BelongsToMany::make('Secondary')->actions(function () {
    return [
        (new Actions\TestPivotAction())->showInline(),
    ];
}),
```

Observe that the action appears when selecting the resource checkbox in the main 'Actions' dropdown menu above the resource table, but doesn't show up in the inline actions menu on the individual resource rows.

**Comment by GerwinV** [comment]
Having the same 'issue'. Now it is not clear enough on a belongstomany relationship that there is an action available.

**Comment by puzzledmonkey** [comment]
Closing due to apathy
